src/Arduboy.h

```diff
@@ -41,17 +41,35 @@ class Arduboy : public Print, public ArduboyCore
    */
   boolean notPressed(uint8_t buttons);
 
-  /// Initializes the hardware
+  /// Initialize hardware, boot logo, boot utilities, etc.
   void begin();
-  /// Initializes the hardware (but with no boot logo)
-  void beginNoLogo();
+
+  /// Init just hardware, no logo, no boot utilities.
+  /**
+   * Look at the source for `begin()` and just rip out what you do not
+   * need and start there.  Calling just `boot()` might work also
+   * depending on your requirements.
+   *
+   * The minimum recommended `begin` replacement:
+   *
+   *   arduboy.boot()         // raw hardware init
+   *   arduboy.audio.begin()  // if you need audio
+   */
+  // void boot(); // defined in core.cpp
+
```

Comment:
Then I guess the next question is "Do we, or do we intend to, use [Doxygen](http://www.stack.nl/~dimitri/doxygen/) to automatically generate documentation for the libaray?". The comments for functions in the header file have been formatted for it.